autoload/neomake/makers/ft/glsl.vim

```diff
@@ -0,0 +1,17 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#glsl#EnabledMakers() abort
+    return executable('glslc') ? ['glslc'] : []
+endfunction
+
+function! neomake#makers#ft#glsl#glslc() abort
+    return {
+        \ 'args': ['-c', '-o', g:neomake#compat#dev_null],
+        \ 'errorformat':
+            \ '%f:%l: %trror: %m,' .
+            \ '%f: %trror: %m,' .
+            \ '%f:%l: %tarning: %m,' .
+            \ '%f: %tarning: %m,' .
+            \ '%-G%s',
+        \ }
+endfunction
```

Comment:
Do you have some example output, which could be added as a test then?
For examples search for `NeomakeTestsGetMakerWithOutput` in `tests`, e.g. https://github.com/blueyed/neomake/blob/8b7816744ed53f32f460a50dc5431f5d13d081c7/tests/ft_python.vader#L118-L129.
Happy to help if you do not want to do it yourself.